source/modules/src/main/sql/database-modules/grid_to_nature/03-views.sql

```diff
@@ -0,0 +1,174 @@
+/*
+ * receptors_to_habitats_view
+ * --------------------------
+ * View on receptors_to_critical_deposition_areas returning only the type 'habitat', using habitat_type_id.
+ */
+CREATE OR REPLACE VIEW grid.receptors_to_habitats_view AS
+SELECT
+	assessment_area_id,
+	critical_deposition_area_id AS habitat_type_id,
+	receptor_id,
+	surface,
+	receptor_habitat_coverage
+
+	FROM grid.receptors_to_critical_deposition_areas
+
+	WHERE type = 'habitat'
+;
+
+
+/*
+ * receptors_to_relevant_habitats_view
+ * -----------------------------------
+ * View on receptors_to_critical_deposition_areas returning only the type 'relevant_habitat', using habitat_type_id.
+ */
+CREATE OR REPLACE VIEW grid.receptors_to_relevant_habitats_view AS
+SELECT
+	assessment_area_id,
+	critical_deposition_area_id AS habitat_type_id,
+	receptor_id,
+	surface,
+	receptor_habitat_coverage
+
+	FROM grid.receptors_to_critical_deposition_areas
+
+	WHERE type = 'relevant_habitat'
+;
+
+
+/*
+ * receptors_to_assessment_areas_on_critical_deposition_area_view
+ * --------------------------------------------------------------
+ * View reducing receptors_to_critical_deposition_areas to a link between assessment areas and receptors.
+ * This only includes receptors within a critical deposition area.
+ * The type of critical deposition area must be filtered by 'type'.
+ * Surface and weight are based on the assessment area/receptor relation (critical deposition areas are combined).
+ */
+CREATE OR REPLACE VIEW grid.receptors_to_assessment_areas_on_critical_deposition_area_view AS
+SELECT
+	assessment_area_id,
+	receptor_id,
+	type,
+	SUM(surface * receptor_habitat_coverage / 10000.0)::real AS weight,  --coverage meenemen
```

Comment:
this trailing comment is also present in the original version, but doesn't make too much sense?

Reply:
The module `grid_terrain_properties` is removed.